_stbt/cv2_compat.py

```diff
@@ -14,7 +14,7 @@
 version = LooseVersion(cv2.__version__).version
 
 if version >= [3, 2, 0]:
-    def find_contour_boxes(image, mode, method):
+    def find_contour_boxes(image, mode, method):  # pylint:disable=redefined-outer-name
```

Comment:
Yeah, it's just pylint getting confused. It's really odd because `function` and `method` aren't builtins in either Python 2 nor 3, and I can't find them mentioned in python-future's `builtins` module.